package.json

```diff
@@ -32,11 +32,11 @@
     "clean": "rm -rf dist/*",
     "prepare": "npm run build",
     "check:client:tsc": "npx tsc --p client/tsconfig.json --noEmit",
-    "build:server": "npx tsc --build server/tsconfig.json",
-    "build:client": "node ./scripts/build.js",
+    "build:server": "npx tsc --build tsconfig.json",
+    "build:client": "node ./scripts/build.mjs",
     "build:styles": "npx sass --no-source-map assets:dist/assets",
     "build": "npm-run-all -s clean check:client:tsc -p build:server build:client build:styles",
-    "lint:script": "npx eslint --ext .ts --ext .js .",
+    "lint:script": "eslint",
```

Comment:
Why do you removed the `npx` call?
Using `npx` ensure that it is using the eslint version in the `node_modules` directory, and not any globally installed `eslint` (which could cause some issue if the version is not matching).